megatron/model/language_model.py

```diff
@@ -256,8 +257,8 @@ def forward(self, input_ids, position_ids, tokentype_ids=None):
 
         # Dropout.
         if self.sequence_parallel:
-            # already partition sequence, do not need scatter_to_sequence_parallel_region
-            # embeddings = tensor_parallel.scatter_to_sequence_parallel_region(embeddings)
+            # already partition sequence, do not need scatter_to_sequence_parallel_region ?
+            embeddings = tensor_parallel.scatter_to_sequence_parallel_region(embeddings)
```

Comment:
Could you please explain why `scatter_to_sequence_parallel_region` is needed?

Reply:
this does splitting on sequence length dimension for sequence parallel otherwise this feature will not work